VisualPinball.Unity/VisualPinball.Unity/VPT/ItemBehavior.cs

```diff
@@ -12,12 +13,15 @@
 
 namespace VisualPinball.Unity.VPT
 {
-	public abstract class ItemBehavior<TItem, TData> : MonoBehaviour, IEditableItemBehavior where TData : ItemData where TItem : Item<TData>, IRenderable
+	public abstract class ItemBehavior<TItem, TData> : MonoBehaviour, IEditableItemBehavior
+		where TData : ItemData where TItem : Item<TData>, IRenderable
 	{
 		[SerializeField]
 		public TData data;
 
 		public TItem Item => _item ?? (_item = GetItem());
+		public bool IsLocked { get { return data.IsLocked; } set { data.IsLocked = value; } }

```

Comment:
Detail, but this is nicer:

```cs
public bool IsLocked { get => data.IsLocked; set => data.IsLocked = value; }
```

Reply:
Just saw it was there before actually.

Reply:
Yeah, just moved it up but i'll change it while I'm in there